plugin/pkg/auth/authenticator/token/oidc/oidc.go

```diff
@@ -20,48 +20,117 @@ package oidc
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"errors"
 	"fmt"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
 
 	"github.com/coreos/go-oidc/jose"
+	"github.com/coreos/go-oidc/oauth2"
 	"github.com/coreos/go-oidc/oidc"
 	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/auth/authenticator"
+	"k8s.io/kubernetes/pkg/auth/authenticator/bearertoken"
 	"k8s.io/kubernetes/pkg/auth/user"
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/net"
 )
 
+const (
+	PathExchangeRefreshToken = "/oidc-exchange-refresh-token"
```

Comment:
I'd like to see related URLs grouped under a common context rather than lots of top-level handlers (e.g. /auth/oidc/...)